initfiles/etc/DIR_NAME/genenvrules.conf

```diff
@@ -1,7 +1,7 @@
 
 [Algorithm]
 max_comps_per_node=4
-do_not_generate=SiteCertificate,dfuplus,soapplus,eclplus,ldapServer,ws_account,eclserver,ecldirect,DynamicESDL,ESPLoggingAgent,LoggingManager,WsLogging
+do_not_generate=SiteCertificate,dfuplus,soapplus,eclplus,ldapServer,ws_account,eclserver,ecldirect,DynamicESDL,esploggingagent,loggingmanager,wslogging
```

Comment:
No.  DynamicESDL is capitalized in the buildset:  https://github.com/hpcc-systems/HPCC-Platform/blob/candidate-6.0.0/initfiles/componentfiles/configxml/buildsetCC.xml.in#L323